apps/speed.c

```diff
@@ -155,8 +155,7 @@ static int usertime = 1;
 
 static double Time_F(int s);
 static void print_message(const char *s, long num, int length, int tm);
-#if (!defined(OPENSSL_NO_RSA) && !defined(OPENSSL_NO_DEPRECATED_3_0)) \
-    || (!defined(OPENSSL_NO_DSA) && !defined(OPENSSL_NO_DEPRECATED_3_0)) \
+#if !defined(OPENSSL_NO_DEPRECATED_3_0) || !defined(OPENSSL_NO_DSA) \
```

Comment:
Wouldn't this normally be broken up differently?  Four lines?

Reply:
Uhm, not necessarily?  Considering this is just a series of simple expressions with || in between, I don't see it as complex enough to need that sort of breaking up